calico_containers/tests/st/test_add_container.py

```diff
@@ -0,0 +1,25 @@
+from sh import ErrorReturnCode
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestAddContainer(TestBase):
+    def test_add_container(self):
+        host = DockerHost('host')
+
+        calicoctl = "/code/dist/calicoctl %s"
+
+        host.execute("docker run -tid --name=node busybox")
```

Comment:
It's not clear the difference between `host.execute` and `host.listen`.  I think the comments for them are out of date.